Extensions/ArtifactEngine/Providers/cacheProvider.ts

```diff
@@ -0,0 +1,108 @@
+import * as fs from 'fs';
```

Comment:
format imports in the following order:
node imports

external library imports

internal class imports